docker/docker-compose.yml

```diff
@@ -63,8 +63,12 @@ services:
     depends_on:
       - db
       - maildev
-    build: .
-    image: jetpack_wordpress:localbuild
+    build:
+      context: .
+      args:
+        PHP_VERSION: ${DOCKER_PHP_VERSION}
```

Comment:
We could also add default to this when constant doesn't exist: `${DOCKER_PHP_VERSION:-7.2-apache}` 
— thus not needing `.env` file in the root folder?

Reply:
We doe have a default in the Dockerfile here:

https://github.com/Automattic/jetpack/pull/9252/files#diff-ebacf6f6ae4ee68078bb16454b23247dR1

Reply:
Now that I think about it, it doesn't seem like we need the `.env` file at all in the end for this thing given that the `yarn docker:up:version` commands will be setting the env var

Reply:
> Now that I think about it, it doesn't seem like we need the `.env` file at all for this thing given that the `yarn docker:up:version` commands will be setting the env var

If you take out the `.env` and don't have a default defined in `docker-compose.yml`, you'll see a warning:
```
WARNING: The DOCKER_PHP_VERSION variable is not set. Defaulting to a blank string.
```

Things would still work but defining a default would take care of the warning:
```yml
    build:
      context: .
      args:
        PHP_VERSION: ${DOCKER_PHP_VERSION:-7.0-apache}
    image: jetpack_wordpress:${DOCKER_PHP_VERSION:-7.0-apache}
```

I wonder what happens to other `docker-compose` commands if PHP version change isn't persistent in `.env` file.
If after initial `yarn docker:up:version` if you want to run any other command, wouldn't they pick the default PHP version and re-build the image?

Reply:
yeah you're right. I think I didn't prepend an evn to the regular `yarn docker:up` either. I like your solution

Reply:
Adding to this that e.g. `yarn docker:clean` script failed for me after playing with versions:

> ERROR: Failed to remove image for service wordpress: 404 Client Error: Not Found ("No such image: jetpack_wordpress:7.0")